### ISSUE TITLE
How to make training data? 

### ISSUE THREAD

**Issue by eeric** [alexjc/neural-enhance#66]
For example,
how much is the size of the training sample? Contains ambiguous images? Contains clear images? Is there a label for the sample?

**Comment by alexjc** [comment]
The dataset is 1M images from the OpenImages dataset: https://github.com/openimages/dataset
Use this script to download them (disable resize): https://github.com/ejlb/google-open-image-download

You can also train with 50k images just fine.

**Comment by eeric** [comment]
Thanks, download as following,
125  138  175  177  19  198  199  31  32  367  379  447  475 ... ...
for 125 folder, it includes 000020780ccee28d.jpg ... ...
As shown above, training samples are like theos, are they?
it doesn't need lable, does it?
in addition, size of training sample isn't same to scale, is it?

**Comment by alexjc** [comment]
No labels are needed, the files are just photos. You can use any photo size, make sure the script doesn't rescale them!

**Comment by jacktive** [comment]
There are only <code>args.epochs</code> * <code>args.epoch-size</code> image fragments using in the trainning,not all the images will be used?
